iree/vm/ops.h

```diff
@@ -57,6 +57,24 @@ static inline int32_t vm_shr_i32u(int32_t operand, int8_t amount) {
   return (int32_t)(((uint32_t)operand) >> amount);
 };
 
+//===------------------------------------------------------------------===//
+// Comparison ops
+//===------------------------------------------------------------------===//
+
+static inline bool vm_cmp_eq_i32(int32_t a, int32_t b) {
+  return a == b ? 1 : 0;
```

Comment:
```suggestion
static inline bool vm_cmp_eq_i32(int32_t lhs, int32_t rhs) {
  return lhs == rhs ? 1 : 0;
```

I would prefer to make the functions look more consistent, meaning to use `lhs` and `rhs`. In addition, we could be closer to the original `(((type)lhs)op((type)rhs)) ? 1 : 0;` pattern, meaning to only skip superfluous `(type)` casts. This applies to the functions below as well.

Reply:
Fixed with latest push